fastapi-challenges/file-upload-api/devashmit-day28/main.py

```
@@ -0,0 +1,33 @@
+ROOT = Path("uploads"); ROOT.mkdir(exist_ok=True)
+MAX = 5*1024*1024; ALLOWED = {"image/jpeg","image/png","image/webp"}
+SIZES = {"thumbnail":(150,150),"medium":(600,600),"large":(1200,1200)}
+app.mount("/images", StaticFiles(directory=str(ROOT)), name="images")
+
+class UpRes(BaseModel): original_name:str; thumbnail_url:str; medium_url:str; large_url:str
+
+@app.post("/upload", response_model=UpRes, status_code=201)
+async def upload(file: UploadFile):
+    if file.content_type not in ALLOWED: raise HTTPException(415, "JPEG/PNG/WebP only")
+    data = await file.read()
+    if len(data) > MAX: raise HTTPException(413, "Max 5MB")
```

Ruff (0.15.18)

[error] Multiple statements on one line (colon)

(E701)